packages/components/forma-36-react-datepicker/tsconfig.json

```diff
@@ -20,8 +20,9 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
+    "allowJs": true,
     "baseUrl": "./",
-    "typeRoots": ["node_modules/@types", "./typings.d.ts"],
+    "typeRoots": ["../../../node_modules/@types", "./typings.d.ts"],
```

Comment:
So this now listens to the global types only? what if someone adds a package only for this component, will the types be available?

Reply:
I'm not sure why we need this tbh, but I need this to get the types because the compiler was complaining and I want to focus on one refactor at a time
I even thought about unifying the tsconfigs in a later PR

What I can say to you is that when adding a dependency for only one package this dependency will be added in the local node_modules, but a dependency used by more than a package is hoisted to the root by yarn-workspaces

if know more about this `typeRoots`, pls tell me :) I can use help

Reply:
Having a base `tsconfig` file is ideal, and then we can extend it for packages if needed. Let's pair on this tomorrow, and get it cleaned up 👍

Reply:
I totally removed this typeRoots and everything seems to be fine actually 👀 

or it will explode something that I'm missing

Reply:
@gui-santos sounds good, if the build is passing then it's reading it properly 👍 
We will look into unifying the tsconfig in another PR <3